csss-site/src/csss/static/csss_static/mm_hackathon/logo-padded.svg

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" standalone="no"?>
```

Comment:
this should be under the location `csss-site/src/events/static/events_static/mm_hackathon/logo-padded.svg`